pkgs/by-name/ne/neovim-node-client/package.nix

```diff
@@ -32,12 +34,23 @@ buildNpmPackage rec {
     ln -s $out/lib/node_modules/neovim/node_modules/.bin/neovim-node-host $out/bin/neovim-node-host
   '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "${placeholder "out"}/bin/neovim-node-host";
```

Comment:
Is this use of placeholder correct?

Reply:
Just not familiar with it and didn't see it defined anywhere in the file, though my have missed it on mobile. Wasn't sure if it was an actual placeholder for something mean to change. If it's as intended no worries.